### ISSUE TITLE
Alt text for WCAG badge

### ISSUE THREAD

**Issue by Tajana-Somogyi** [ALASdoo/accessliaison#65]
This is the code that W3C says should be used for the badge:

<a href="https://www.w3.org/WAI/WCAG2AA-Conformance"
   title="Explanation of WCAG 2 Level AA conformance">
  <img height="32" width="88"
       src="https://www.w3.org/WAI/WCAG22/wcag2.2AA-blue"
       alt="Level AA conformance,
            W3C WAI Web Content Accessibility Guidelines 2.2">
</a>
